src/server/scripts/Northrend/Ulduar/Ulduar/ulduar.h

```diff
@@ -368,7 +399,7 @@ enum UlduarData
     DATA_HODIR_YS,
     DATA_THORIM_YS,
     DATA_MIMIRON_YS,
-    DATA_ILLUSION,
+    DATA_ILLUSION, // = 46 used by conditions
```

Comment:
would be better to explicitly set the value then (instead of just leaving a comment)